### ISSUE TITLE
Example for adding custom templates and layouts

### ISSUE THREAD

**Issue by eshaiju** [owen2345/Camaleon-CMS-Sample#3]
Please add a custom template and layout also like to know how to integrate custome design rather than a template

**Comment by cmckni3** [comment]
:+1:

**Comment by asn** [comment]
yes this would be very nice. How to handle assets with a template. what is the preferred way to integrate custom css and js?

**Comment by owen2345** [comment]
This functionality is supported by Camaleon CMS V2.
http://camaleon.tuzitio.com/
About assets you can manage like rails assets.

```
<%= javascript_include_tag..... %>
```
